core/src/main/java/com/graphhopper/routing/Router.java

```diff
@@ -380,7 +380,7 @@ protected void checkProfileCompatibility() {
 
         protected abstract Weighting createWeighting();
 
-        protected EdgeFilter getSnappingFilter() {
+        protected EdgeFilter getSnapFilter() {
             return new DefaultSnapFilter(weighting, lookup.getBooleanEncodedValue(Subnetwork.key(profile.getName())));
```

Comment:
I stumbled now twice over this inconsistent naming and had to change it ;) (feel free to change DefaultSnapFilter to DefaultSnappingFilter instead :) )

Reply:
No no either way is fine for me